cmd/server/main.go

```diff
@@ -415,10 +415,12 @@ func (p *Proxy) runUDSMasterServer(ctx context.Context, o *ProxyRunOptions, s *s
 		grpcServer := grpc.NewServer()
 		client.RegisterProxyServiceServer(grpcServer, s)
 		var lc net.ListenConfig
+		oldUmask := syscall.Umask(0007)
 		lis, err := lc.Listen(ctx, "unix", o.udsName)
 		if err != nil {
 			return nil, fmt.Errorf("failed to listen(unix) name %s: %v", o.udsName, err)
 		}
+		syscall.Umask(oldUmask)
```

Comment:
This isn't going to be reached if the Listen fails.

Reply:
:man_facepalming: Fixed. Thanks for pointing this out.